rasa/core/policies/ted_policy.py

```diff
@@ -535,6 +535,8 @@ def __init__(
 
         self.all_labels_embed = None  # needed for efficient prediction
 
+        self._prepare_layers()
```

Comment:
`prepare_layers` is called inside `TransformerRasaModel`

Reply:
same as below :)